### PR TITLE
Add script for generating grammars.dat and setup CI

### PR DESCRIPTION
**Pull request by STmihan**
Rewrite generate.js a bit to make it work in node js, using some `eval` magic from [here](https://stackoverflow.com/a/23699187/17140794). To see the most important changes in it check out https://github.com/STmihan/libprisma/commit/bde28aa8e798a25ba93beedd018800822ddc198c

I set up the workflow for publishing as an action artifact [example here](https://github.com/STmihan/libprisma/actions/runs/6600640871), but I right now looking on a https://github.com/TelegramMessenger/libprisma/pull/1 and it looks cool too.
If you think that [MarshalX's](https://github.com/MarshalX) variant is better, I can just add his PR workflow to my solution.

### CODE REVIEW

**Comment by STmihan** [comment]
Okay, I just copied the workflow from #1, but I am not sure how to check that it does anything (because both `grammars.dat` are the same)

**Comment by FrayxRulez** [comment]
Hey, this is exactly what I wanted to have.
I would say that it makes sense to just rename `generate-node.js` to `generate.js` and keep just this code :)
Only one thing I'd like to change: `isEqual.js` actually comes from a `npm` package: https://github.com/lodash/lodash/tree/4.5.0-npm-packages/lodash.isequal
Could you change your code to actually install and use that rather than the local copy? Thank you!

**Comment by STmihan** [comment]
Done. I have created an npm repo, added "lodash.isequal" as a dependency, and updated README.md from the master

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "libprisma",
+  "version": "1.0.0",
+  "description": "This is a C++ porting of [prism.js](https://github.com/PrismJS/prism) library.\r The code depends on Boost.Regex, as it's a faster and more comprehensive than STD's.",
+  "main": "generate-node.js",
```
Comment (FrayxRulez) [inline package.json:5]:
```suggestion
  "main": "generate.js",
```

**Review by FrayxRulez — CHANGES_REQUESTED** [review]

**Comment by Phi242** [comment]
A

**Review by Phi242 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Phi242) [reply package.json]: Yy

**Review by Phi242 — CHANGES_REQUESTED** [review]
Fh

**Pull request merged**